earth_enterprise/src/fusion/rasterfuse/TmeshPackgenTraverser.cpp

```diff
@@ -150,6 +157,15 @@ TmeshPackgenTraverser::TmeshPackgenTraverser(
       writer_(file_pool_, output, true /* overwrite */),
       extra_cache_(20000 /* apx 160M */),
       not_covered_tiles_exist_(false) {
+  if(decimation_threshold == 0.0)
```

Comment:
should we handle invalid decimation_threshold values as either an error or keep processing as 'no decimation' ? I noticed this parameter validation is done in the command line that end-up calling this class, but it would be also to have validation at this entry point since it may be called from other clients (that might not did validation) ...

Reply:
So you think I should check to see if decimation_threshold is <= 0.0  here?

Reply:
It does not hurt since any client code calling this might not check.